### ISSUE TITLE
make a different wallpaper for each workspace

### ISSUE THREAD

**Issue by Sim4n6** [UtkarshGpta/bing-desktop-wallpaper-changer#26]

**Comment by truongan** [comment]
Is that even possible ?

**Comment by Sim4n6** [comment]
I'm just suggesting...

**Comment by truongan** [comment]
I mean is that even possible to manually set different wallpaper for each workspace?  The script can only do things that people can do themself you know.

**Comment by Sim4n6** [comment]
Yes, [from askUbuntu](http://askubuntu.com/questions/75998/is-it-possible-to-have-a-different-background-for-each-workspace).

**Comment by truongan** [comment]
Is it possible on something newer on 14.04? It's 2 years old already.

**Comment by Sim4n6** [comment]
same concept for Ubuntu 16.04.1, please check this [forum link](https://forums.linuxmint.com/viewtopic.php?t=118540#p672189)

**Comment by truongan** [comment]
I see nothing in your link regarding ubuntu 16.04. They were talking about KDE and about can Mate/Cinamon do that. The last dicussion end futily in 2015

**Comment by vishal758** [comment]
If u are using version 14.04 u may try it 
http://ubuntuhandbook.org/index.php/2014/04/ubuntu-14-04-set-different-wallpaper-for-each-workspace/
